physics/src/modules/MU71Albedo.cpp

```diff
@@ -0,0 +1,33 @@
+/*!
+ * @file MU71Albedo.cpp
+ *
+ * @date Wed 24 Aug 2022 08:05:49 CEST
+ * @author Einar Örn Ólason <einar.olason@nersc.no>
+ */
+
+#include "include/MU71Albedo.hpp"
+
+namespace Nextsim {
+
+MU71Albedo::MU71Albedo()
+    : snowAlbedo(albedoTable)
+{
+}
+
+std::tuple<double, double> MU71Albedo::albedo(double temperature, double snowThickness, double i0)
```

Comment:
You're right that ``albedo`` is not a descriptive enough name, but it's there because returning ``penSW``, as well as ``albedo``, is an afterthought. Something like ``surfaceSWBalance`` or ``surfaceShortWaveBalance`` would be more correct because the ``doubles`` ``albedo`` and ``penSW`` are the fraction of the incoming shortwave radiation that is reflected into space and transmitted into the interior of the ice, respectively.

The problem is that both names are confusing; ``albedo`` returns more than albedo, which is confusing, and nobody talks about the "surface short-wave balance", so that's also confusing. On the whole, though, I think something like ``surfaceShortWaveBalance`` is probably better, but I'll have a bit of a think to see if I can come up with a better name.